internal/constants/constants.go

```diff
@@ -72,6 +72,9 @@ const (
 	CARootPrivateKey = "ca.key"
 	OldCARootCert    = "old-ca.crt"
 
+	// Client CA ConfigMap.
+	ClientCACert = "client-ca.crt"
```

Comment:
I would rename it to make it obvious that's the CA of the Kubernetes API server client certificate. Just to distinguish it from the CA used by `audit-scanner`

Reply:
I’d keep this variable as it is because it matches the webhook option name in controller-runtime. There's no need to separate audit-scanner and API server keys since the controller doesn’t use the audit-scanner concept and will just inject the internal CA for general use.

Reply:
You're right, good points!